Code/Framework/AzQtComponents/AzQtComponents/Components/resources.qrc

```diff
@@ -321,6 +321,7 @@
     <file>img/logging/copy.svg</file>
     <file>img/logging/reset.svg</file>
     <file>img/logging/add-filter.svg</file>
+    <file>img/logging/failure.svg</file>
```

Comment:
should Code/Framework/AzQtComponents/AzQtComponents/Components/img/logging/warning.svg be here too?